gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-oidc/src/main/java/io/gravitee/am/gateway/handler/oauth2/resources/auth/provider/CertificateUtils.java

```diff
@@ -61,6 +61,11 @@ public static Optional<X509Certificate> extractPeerCertificate(RoutingContext ro
 
         if (certHeaderValue != null) {
             try {
+                certHeaderValue = certHeaderValue
+                        .replaceAll("\\+","%2F")
+                        .replaceAll("//","%2B")
+                        .replaceAll("=","%3D");
+
```

Comment:
could you also chain with `replaceAll("\t", "\n")` from following line?

Reply:
Changed.